operate/qa/integration-tests/src/test/java/io/camunda/operate/webapp/BackupControllerIT.java

```diff
@@ -68,20 +65,21 @@
 import org.springframework.test.context.ActiveProfiles;
 import org.springframework.test.context.junit4.SpringRunner;
 
+/* This class should be moved to the dist/ module as it does not really test anything operate specific
```

Comment:
:question: This is actually valid format for Javadocs? :smile:

Reply:
Well it was not meant to be a Javadocs, just a temporary comment